ksql-rest-app/src/test/java/io/confluent/ksql/rest/server/computation/CommandRunnerTest.java

```diff
@@ -16,80 +16,90 @@
 
 package io.confluent.ksql.rest.server.computation;
 
-import org.apache.kafka.clients.consumer.KafkaConsumer;
-import org.apache.kafka.clients.producer.KafkaProducer;
-import org.apache.kafka.common.serialization.Deserializer;
-import org.apache.kafka.common.serialization.Serializer;
+import org.apache.kafka.clients.consumer.ConsumerRecord;
+import org.apache.kafka.clients.consumer.ConsumerRecords;
+import org.apache.kafka.common.TopicPartition;
 import org.junit.Assert;
 import org.junit.Test;
 
+import java.util.ArrayList;
+import java.util.Collections;
 import java.util.HashMap;
+import java.util.List;
 import java.util.Map;
 
-import io.confluent.kafka.serializers.KafkaJsonDeserializer;
-import io.confluent.kafka.serializers.KafkaJsonSerializer;
-import io.confluent.ksql.metastore.MetaStoreImpl;
 import io.confluent.ksql.rest.entity.CommandStatus;
-import io.confluent.ksql.rest.server.StatementParser;
-import io.confluent.ksql.rest.server.mock.MockCommandStore;
-import io.confluent.ksql.rest.server.mock.MockKafkaTopicClient;
-import io.confluent.ksql.rest.server.mock.MockKsqkEngine;
 import io.confluent.ksql.rest.server.utils.TestUtils;
+import io.confluent.ksql.util.Pair;
+
+import static org.easymock.EasyMock.*;
 
 public class CommandRunnerTest {
 
+  /**
+   *   Get a command runner instance using mock values
+    */
   private CommandRunner getCommanRunner(StatementExecutor statementExecutor) {
+    List<Pair<CommandId, Command>> commandList = new TestUtils().getAllPriorCommandRecords();
+    List<ConsumerRecord<CommandId, Command>> recordList = new ArrayList<>();
+    for (Pair commandPair: commandList) {
+      recordList.add(new ConsumerRecord<CommandId, Command>("T", 1, 1, (CommandId) commandPair
+          .getLeft(), (Command) commandPair.getRight()));
+    }
+    Map<TopicPartition, List<ConsumerRecord<CommandId, Command>>> recordMap = new HashMap<>();
+    recordMap.put(new TopicPartition("T", 1), recordList);
+    CommandStore commandStore = mock(CommandStore.class);
+    expect(commandStore.getNewCommands()).andReturn(new ConsumerRecords<>(recordMap));
+    expect(commandStore.getPriorCommands()).andReturn(Collections.emptyList());
+    replay(commandStore);
+    return new CommandRunner(statementExecutor, commandStore);
+  }
 
-    Map<String, Object> commandConsumerProperties = new HashMap<>();
-    commandConsumerProperties.put("bootstrap.servers", "localhost:9092");
-    Serializer<Command> commandSerializer = new KafkaJsonSerializer<>();
-    Deserializer<Command> commandDeserializer = new KafkaJsonDeserializer<>();
-    Serializer<CommandId> commandIdSerializer = new KafkaJsonSerializer<>();
-    Deserializer<CommandId> commandIdDeserializer = new KafkaJsonDeserializer<>();
+  private StatementExecutor getMockStatementExecutor() throws Exception {
```

Comment:
This method is not needed. As all the `getStatuses` method is not called by `CommandRunner` so instead the test methods can just create the `StatementExecutor` mock directly

Reply:
Actually we don't even need any assertion in this test and just need to make sure the corresponding methods in the mock objects are called. So just using verify would be enough. Refactoring the test accordingly.